llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -10389,8 +10390,9 @@ PreservedAnalyses LoopVectorizePass::run(Function &F,
       PA.preserve<DominatorTreeAnalysis>();
       PA.preserve<ScalarEvolutionAnalysis>();
 
-#ifdef EXPENSIVE_CHECKS
-      SE.verify();
+#ifndef NDEBUG
+      if (VerifySCEV)
```

Comment:
Not sure if interleaving this with the different `PA.preserve` is the best place. Might make sense to move it to `runImpl`, perhaps to the same block as `LAIs->clear()`